.github/workflows/build.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         env:
           VAULT_DEV_ROOT_TOKEN_ID: root
           VAULT_LICENSE: ${{ secrets.VAULT_LICENSE }}
+          VAULT_AUTH_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I believe this should be in the `steps` section since it's not an environment variable key to the Vault server setup. We already have a `GITHUB_TOKEN` env variable being injected on [L131](https://github.com/hashicorp/terraform-provider-vault/pull/1763/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L130), I believe if the Vault PR gets merged then we will have to update the environment varibale